src/api_manager/request_handler.cc

```diff
@@ -29,6 +29,22 @@ using google::devtools::cloudtrace::v1::Traces;
 namespace google {
 namespace api_manager {
 
+namespace {
+// The header key to send endpoint api user info.
+const char kEndpointApiUserInfo[] = "X-Endpoint-API-UserInfo";
```

Comment:
There is one defined in check_auth.cc and check_auth_test.cc
Could you put them into one definition?

Reply:
Moved to google::api_manager::auth::kEndpointApiUserInfo